### PR TITLE
Improvements to stitch script

### PR DESCRIPTION
**Pull request by IcyMidnight**
- Support python3
- Add better controls
- Fix no-capture mode

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -8,17 +8,56 @@ that were taken.
...
+                                 [--max-captures MAX_CAPTURES]
+
+Take Wear screenshots using adb and stitch them together
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --out-dir OUT_DIR     The dirctory to output to. (default: the current
+                        directory)
+  --file-prefix FILE_PREFIX
+                        The file prefix to use. An autoicrementing index is
```
Comment (ryanl) [inline README]: typo (autoicrementing)

File: README [hunk 2]
```diff
@@ -8,17 +8,56 @@ that were taken.
...
+                        added to generate the full filename so the previous
+                        captures are not overwritten. Mutally exclusive with
```
Comment (ryanl) [inline README]: typo (mutally)

File: README [hunk 3]
```diff
@@ -8,17 +8,56 @@ that were taken.
...
+                        wear_screenshot_stitch.py --round --adb-args " -e"
+
+Capture options:
+  --capture             Capture new images to stitch. Contrast with --no-
+                        capture. (default)
+  --no-capture          Do not capture new images, just try to stitch existing
+                        images. Contrast with --capture.
+  --round               Set capture type for round displays, framing the
+                        stitched image with round borders. Contrast with
+                        --sqaure. (default)
```
Comment (ryanl) [inline README]: typo (sqaure)

File: wear_screenshot_stitch.py [hunk 4]
```diff
@@ -74,49 +79,102 @@ def get_row_hashes(image_id):
     row_hashes.append(row_hash)
   return row_hashes
 
+def padded_index(max, num):
+  index_chars = int(math.ceil(math.log10(max)))
```
Comment (ryanl) [inline wear_screenshot_stitch.py:83]: Recommend avoiding float math -- rounding can cause subtle bugs.

Does str(num).zfill(len(str(max)) work instead?

File: README (hunk 1 above)
Comment (IcyMidnight) [reply README]: Fixed

**Review by ryanl — APPROVED** [review]

File: wear_screenshot_stitch.py (hunk 4 above)
Comment (IcyMidnight) [reply wear_screenshot_stitch.py:83]: No because we're 0-based so len(100) will result in indexes of 000-099 which is silly.

File: README (hunk 3 above)
Comment (IcyMidnight) [reply README]: Fixed

File: README (hunk 2 above)
Comment (IcyMidnight) [reply README]: Fixed

**Pull request merged**